lib/src/authentication/sign_in/sign_in_view.dart

```diff
@@ -0,0 +1,143 @@
+import "package:flutter/foundation.dart" show defaultTargetPlatform;
+import "package:flutter/material.dart";
+import "package:flutter_gen/gen_l10n/app_localizations.dart";
+import "package:go_router/go_router.dart";
+import "package:kana_to_kanji/src/authentication/landing_view.dart";
+import "package:kana_to_kanji/src/authentication/sign_in/sign_in_view_model.dart";
+import "package:kana_to_kanji/src/authentication/widgets/button_apple_logo.dart";
+import "package:kana_to_kanji/src/authentication/widgets/button_google_logo.dart";
+import "package:kana_to_kanji/src/authentication/widgets/input_email.dart";
+import "package:kana_to_kanji/src/authentication/widgets/input_password.dart";
+import "package:kana_to_kanji/src/core/widgets/app_scaffold.dart";
+import "package:stacked/stacked.dart";
+
+class SignInView extends StatefulWidget {
+  static const routeName = "/authentication/login";
+
+  const SignInView({
+    super.key,
+  });
+
+  @override
+  State<SignInView> createState() => _SignInViewState();
+}
+
+class _SignInViewState extends State<SignInView> {
+  bool passwordVisible = true;
+
+  @override
+  void initState() {
+    super.initState();
+    passwordVisible = true;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final AppLocalizations l10n = AppLocalizations.of(context);
+
+    final Widget textDivider = Expanded(
+      child: Padding(
+        padding: const EdgeInsets.only(left: 20.0, right: 20.0),
+        child: Divider(
+          color: Theme.of(context).colorScheme.primaryContainer,
+        ),
+      ),
+    );
+
+    return ViewModelBuilder<SignInViewModel>.reactive(
+      viewModelBuilder: () => SignInViewModel(GoRouter.of(context)),
+      builder: (context, viewModel, child) => AppScaffold(
+        appBar: AppBar(
+          backgroundColor: Colors.transparent,
+          leading: InkWell(
```

Comment:
Why use that? `AppBar.automaticallyImplyLeading` should automatically show a back arrow, and if it's not the case, we should use an `IconButton`

Reply:
C'est changé